tests/NPSP.py

```diff
@@ -360,7 +393,12 @@ def select_lightning_checkbox(self,title):
     def select_lightning_table_checkbox(self,title):
         """"""
         locator=npsp_lex_locators['table_checkbox'].format(title)
-        self.selenium.get_webelement(locator).click() 
+        self.selenium.get_webelement(locator).click()
+        
+    def select_bge_checkbox(self,title):
+        """"""
```

Comment:
There's no reason to provide an empty docstring. Either actually write a description of what the keyword does / when it is meant to be used, or leave it out altogether.

Reply:
removed empty strings

Reply:
I have created another branch to make code improvements like storing session records for deletion and adding comments so I will be taking care of this in that branch